### ISSUE TITLE
How to handle Access Denied issue

### ISSUE THREAD

**Issue by ThapaMahesh** [commercelayer/gatsby-commerce#2]
I get **Access Denied** error on orders filter[token] request with line items. Maybe the token has expired but how do I handle the issue so I get new token which will have no error hopefully. I am using sales channel with client id and market scope.

The error happened suddenly on working project.

**Comment by fconforti** [comment]
Yes, the sales channel can perform this action. The problem was that we still needed to update the js library after our latest release.

We just pushed a fix. Let me know if everything works as expected.

Thanks,
F.

**Comment by ThapaMahesh** [comment]
I updated the package and ran but it still doesn't work with **sales channel**. I get the same error for request with _filter[token]_ to _orders_
